record/record.go

```diff
@@ -41,10 +42,13 @@ func RemapRecord(inputRecord FluentBitRecord, inputTimestamp interface{}, plugin
 	if !ok {
 		source = "BARE-METAL"
 	}
-	outputRecord["plugin"] = map[string]string{
-		"type":    "fluent-bit",
-		"version": pluginVersion,
-		"source":  source,
+	_, ok = outputRecord["plugin"]
+	if !ok {
```

Comment:
Cool!! You can use this shorter go-ish notation form:
```
if _, ok = outputRecord["plugin"]; !ok {
```

Reply:
Done!